### PR TITLE
pingone_system_application resource export

### PR DESCRIPTION
**Pull request by wesleymccollam**

### CODE REVIEW

File: internal/connector/pingone_platform/resources/pingone_system_application.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+			appIdOk   bool
+			appName   *string
+			appNameOk bool
+		)
+
+		switch {
+		case app.ApplicationPingOnePortal != nil:
+			appId, appIdOk = app.ApplicationPingOnePortal.GetIdOk()
+			appName, appNameOk = app.ApplicationPingOnePortal.GetNameOk()
+			// case app.
```
Comment (erikostien-pingidentity) [inline internal/connector/pingone_platform/resources/pingone_system_application.go:55]: nit: comment

File: internal/connector/pingone_platform/resources/pingone_system_application.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+		case app.ApplicationPingOneSelfService != nil:
+			appId, appIdOk = app.ApplicationPingOneSelfService.GetIdOk()
+			appName, appNameOk = app.ApplicationPingOneSelfService.GetNameOk()
+		}
```
Comment (erikostien-pingidentity) [inline internal/connector/pingone_platform/resources/pingone_system_application.go:59]: Perhaps a default case to continue here if not either.

Comment (wesleymccollam) [reply internal/connector/pingone_platform/resources/pingone_system_application.go:60]: updated

**Review by erikostien-pingidentity — APPROVED** [review]
Looks good!

**Pull request merged**